service/history/queues/executable.go

```diff
@@ -81,7 +81,8 @@ var (
 	// reschedulePolicy is the policy for determine reschedule backoff duration
 	// across multiple submissions to scheduler
 	reschedulePolicy                      = common.CreateTaskReschedulePolicy()
-	taskNotReadyReschedulePolicy          = common.CreateTaskNotReadyReschedulePolicy()
+	taskNotReadyReschedulePolicy          = common.CreateDeleteOpenExecReadyReschedulePolicy()
+	deleteOpenExecutionReschedulePolicy   = common.CreateDeleteOpenExecReadyReschedulePolicy()
```

Comment:
I think the naming is confusing here, workflow execution is already closed, it's just the async processing of the associated close tasks are not completed yet. (also, `Ready` should be removed from the name?)

So maybe something like `dependencyTaskNotCompleted` error or reschedule policy? The error and policy also needs to be reused by the transfer queue later.

Reply:
Renamed everywhere--thanks!